src/components/ShowsList.js

```diff
@@ -0,0 +1,150 @@
+    dispatch(fetchShows());
+  }, [dispatch]);
+
+  if (status === 'loading') {
+    return <div>Loading...</div>;
+  }
+
+  if (status === 'failed') {
+    return (
+      <div>
+        Error:
+        {error}
+      </div>
+    );
+  }
+
+  return (
+    <div className="App">
+
+      <div className="webHeader">
+        <WebHeader />
+      </div>
+      <div className="mobileHeader">
+        <MobileHomeHeader />
+      </div>
+      <input
+        type="text"
+        placeholder="Search shows..."
+        className="webSearch"
+        onChange={(e) => setSearchInput(e.target.value)}
+      />
+
+      <div className="mobileHeaderExtension">
+        <div className="extensionChildMain">
+          <h1>Home</h1>
+          <span>
+            <b>
+              {' '}
+              {totalCount}
+              {' '}
+              <br />
+              {' '}
+              total shows
+              {' '}
+            </b>
+            <b>
+              {' '}
+              {endedShowsCount}
+              {' '}
+              <br />
+              {' '}
+              completed shows
+              {' '}
+            </b>
+            <b>
+              {runningShowsCount}
+              {' '}
+              <br />
+              {' '}
+              ongoing shows
+              {' '}
+            </b>
+          </span>
+        </div>
+
+        <div className="extensionChildSub">
+          <span>
+            <i>Suggested</i>
+            {' '}
+            <a>learn More</a>
+          </span>
+        </div>
+
+      </div>
+
+      <h1 className="webHeader">Home</h1>
+      <ul className="Shows">
+        {filteredShows.map((show) => (
+          <div key={show.id} className="mObjectContainer">
+            <li>
+              <div className="ShowsItems">
+                <img src={show.image?.medium} alt={show.name} />
+                {' '}
+                <br />
+                <div className="ShowsItemsInfo">
+                  <Link to={`/show/${show.id}/seasons`}>
+                    {' '}
+                    <b>
+                      {show.name}
+                      {' '}
+                    </b>
+                  </Link>
+                  {' '}
+                  <br />
+                  <span>
+                    Rating:
+                    <i>
+                      {show.rating.average}
+                      {' '}
+                    </i>
+                  </span>
+                  {' '}
+                  <span>
+                    Show Status:
+                    <i>
+                      {' '}
+                      {show.status}
+                      {' '}
+                    </i>
+                  </span>
+                  <br />
+                  {' '}
+                  <span>
+                    Premier Date:
+                    <i>{show.premiered}</i>
+                  </span>
+                </div>
+              </div>
+            </li>
+          </div>
+        ))}
```

Comment:
- [ ] 👍 Great job so far on your implementation! 🚀 However, I'd like to emphasize the importance of aligning your work with the project requirements. 📜

  - Upon reviewing your `homepage`, it's apparent that the layout doesn't quite match the provided design in the project requirements. 🧐 The requirements specify that we should adhere to the design in terms of `layout`, `typography`, and `colors`, while allowing room for personalizing the content. 

  - To ensure your work aligns with the project design, please make sure that your design choices, especially in terms of colors and layout, closely resemble those depicted in the provided design. 🔍

I've included a screenshot below for a side-by-side comparison to help illustrate the differences:

<table>
<tr>
<th align=“middle”>Given design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="206" alt="Screenshot 2023-09-29 at 14 32 47" src="https://github.com/paulshaun-kdtk/React_Capstone_Metrics/assets/107192076/ada58de9-61e4-4e12-a4f1-e03b51ec9d24">

</td>
<td><img width="335" alt="Screenshot 2023-09-29 at 14 19 15" src="https://github.com/paulshaun-kdtk/React_Capstone_Metrics/assets/107192076/0e3f9d4d-ddef-4fb0-9c9a-39f350e80c9d">
</td>
</tr>
</table>

As you can see from the comparison, there are discrepancies in the colors and layout between your implementation and the given design. While you've integrated some of the colors, there's room for improvement to achieve a more consistent look throughout the entire application. 🌈

Happy coding! 😊🍀🚀